docs/writing-tests/component-testing.mdx

```diff
@@ -49,7 +49,7 @@ The test itself is defined inside a `play` function connected to a story. Here's
 
 {/* prettier-ignore-start */}
 
-<CodeSnippets path="login-form-with-play-function.md" usesCsf3 csf2Path="writing-tests/interaction-testing#snippet-login-form-with-play-function" />
+<CodeSnippets path="login-form-with-play-function.md" usesCsf3 csf2Path="writing-tests/component-testing#snippet-login-form-with-play-function" />
```

Comment:
@JReinhold if you're ok with it, you can remove the `usesCSF3` and `csf2Path` props from the code snippets as they are no longer relevant.